docs/containers/debug-netcore.md

```diff
@@ -19,16 +18,35 @@ MetaDescription: Debug a .NET app running in a Docker container, using Visual St
 
 ## Walkthrough
 
-1. If needed, create a .NET project with `dotnet new`.
-1. Open the project folder in VS Code.
+- If needed, create a .NET project with `dotnet new`.
+- Open the project folder in VS Code.
+- Optionally, set a breakpoint.
+
+## .NET SDK vs. Dockerfile Build
+
+For those seeking a streamlined container debugging experience without the need for a Dockerfile or who are well-acquainted with MSBuild, the .NET SDK option is the ideal choice. On the other hand, if you feel more at ease working with Dockerfiles, opt for Dockerfile.
+
+### .NET SDK Container Build (Debug without `Dockerfile`)
+
+1. Press `kb(workbench.action.debug.start)` or choose **Start Debugging** from the **Run** menu. (Make sure you don't have any existing launch profiles in `launch.json`)
+1. You're going to be prompted with a list of debuggers, choose Select **Docker: Debug in Container**
```

Comment:
```suggestion
1. You're prompted with a list of debuggers. Choose **Docker: Debug in Container**
```

Reply:
Use present tense. Also is the "Select" part of the UI or was it redundant with "choose" ?

Reply:
Good catch, "Select" is indeed redundant